make/autoconf/jdk-version.m4

```diff
@@ -499,15 +501,18 @@ AC_DEFUN_ONCE([JDKVER_SETUP_JDK_VERSION_NUMBERS],
     AC_MSG_ERROR([--with-vendor-version-string must have a value])
   elif [ ! [[ $with_vendor_version_string =~ ^[[:graph:]]*$ ]] ]; then
     AC_MSG_ERROR([--with--vendor-version-string contains non-graphical characters: $with_vendor_version_string])
-  else
+  elif test "x$with_vendor_version_string" != xno; then
+    # Set vendor version string if --without is not passed
+    # Check not required if an empty value is passed, since VENDOR_VERSION_STRING
+    # would then be set to ""
```

Comment:
I'm not sure that works. In general, values like these are *supposed* to have a value. The default fallback is "N/A", which is not the empty string. I see no point in allowing overriding this to be empty. It just introduces cases where the build might fail later, for something that will hardly ever be tested.

Reply:
It's in make/conf/branding.conf.

Reply:
I don't see a VENDOR_VERSION_STRING in there though, all vendor related defaults I could find are
COMPANY_NAME=N/A
VENDOR_URL=https://openjdk.java.net/
VENDOR_URL_BUG=https://bugreport.java.com/bugreport/
VENDOR_URL_VM_BUG=https://bugreport.java.com/bugreport/crash.jsp

Reply:
Ah, sorry, you are right. I mixed it up with COMPANY_NAME.

Yes, VENDOR_VERSION_STRING is allowed to be empty, so your code is correct.